special-pages/pages/release-notes/app/components/ContentPlaceholder.module.css

```diff
@@ -1,9 +1,16 @@
 .contentPlaceholder {
+
     & h2,
     & p {
         background: var(--color-black-at-6);
 
-        @media (prefers-color-scheme: dark) {
+    }
```

Comment:
Not a problem with this change, but with the code I wrote that preceded it. I’d probably rewrite it as 

```
    --placeholder-background-color: var(--color-black-at-6);

    [data-theme=dark] & {
        --placeholder-background-color: var(--color-white-at-6);
    }
```

Then change every instance of `--color-white-at-6` and `--color-black-at-6`. I can do that as a separate PR if your prefer.